src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityBarbarian.java

```diff
@@ -0,0 +1,107 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.entity.EntityCitizen;
+import com.minecolonies.coremod.sounds.BarbarianSounds;
+import net.minecraft.entity.SharedMonsterAttributes;
+import net.minecraft.entity.ai.*;
+import net.minecraft.entity.monster.EntityMob;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.world.World;
+
+import javax.annotation.Nullable;
+
+/**
+ * Created by Asherslab on 5/6/17.
+ */
+public class EntityBarbarian extends EntityMob
+{
+    private final Colony colony = ColonyManager.getClosestColony(world, this.getPosition());
+
+    /* default */ private static final int BARBARIAN_BASE_HEALTH = 25;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)